src/Surfnet/StepupMiddleware/ApiBundle/Identity/Projector/AuditLogProjector.php

```diff
@@ -99,7 +100,12 @@ private function applyAuditableEvent(AuditableEvent $event, DomainMessage $domai
         $entry->identityId          = (string) $auditLogMetadata->identityId;
         $entry->identityInstitution = $auditLogMetadata->identityInstitution;
         $entry->event               = get_class($event);
-        $entry->recordedOn          = new DateTime(new CoreDateTime($domainMessage->getRecordedOn()->toString()));
+        $entry->recordedOn          = new DateTime(
```

Comment:
Ah, just tested this again and it does. I was testing with the "H" formatted dates. Removed this change.